spec/requests/users_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe User do
+  it 'responds with correct status users index' do
+    get '/users'
+    expect(response).to have_http_status(:ok)
+  end
+
+  it 'renders correct template for users index' do
+    get '/users'
+    expect(response).to render_template('index')
+  end
+
+  it 'responds with correct status user show' do
+    get '/users/:id'
+    expect(response).to have_http_status(:ok)
+  end
+
+  it 'renders correct template for user show' do
+    get '/users/:id'
+    expect(response).to render_template('show')
+  end
+end
```

Comment:
- [ ] Good job 👍 working on these test cases. It will be nice to include a test that ensures the response body includes the correct placeholder text.
Example: 
```ruby
expect(response.body).to include("Welcome to user index page")
```

Reply:
Hi @mmsesay

Thanks for pointing that out.

The test for placeholder text is actually in the feature directory.

Reply:
Hello @emmiiorji, thanks for your comments. 
The tests were supposed to be in the `requests` directory but it's good you already have them.

Reply:
@emmiiorji I forgot to mention this, it is also required to specify the type of test you are writing. 
Please see the snippet below as an example to achieve that.

```ruby
RSpec.describe "Users", type: :request do
...
end

RSpec.describe "Posts", type: :request do
...
end
```

Reply:
Okay. Thank you.